fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocksLex.g4

```diff
@@ -57,6 +57,7 @@ CATALOGS: 'CATALOGS';
 CEIL: 'CEIL';
 CHAIN: 'CHAIN';
 CHAR: 'CHAR';
+CHARACTER: 'CHARACTER';
```

Comment:
Should add this to nonReserve in StarRocks.g4

Reply:
I think is no, in this doc https://dev.mysql.com/doc/refman/5.7/en/keywords.html
CHARACTER (R) is Reserved Words.